### DISCUSSION TITLE
Adjusting playback history to keep

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#7364] [Q&A]
### This article found in Azuracast local docs : Optimizing 
--------------------------------------
### Keep Less Playback History
In “System Settings”, you will notice a setting called “Days of Playback History to Keep”. This setting not only controls how far back you can view song history, but also how far back you can view detailed listener statistics.
Setting it to a large value or “Indefinitely” will quickly fill up your available hard drive space. If you change this setting, within an hour or so AzuraCast will automatically “trim” any extra song history and listener data, which will often result in a significant reduction in disk space used.
----------------------------------------
### So how do we set to hours???

**Comment by BusterNeece** [comment]
We don't currently go down to the hour level because many playlists check to see if a track has been played within that timespan, and losing history would cause those checks to have problems. If you don't have any of those checks, you could submit a feature request to have those added: https://features.azuracast.com/

**Reply by SpeedyCharly** [reply]
Most anti-repeats are set to 60 or 120 minutes in my (limited) experience.  Big span for improvement between that and 14 days. Bottom setting could be something like 24 hrs to help those with limited storage space... 

But ok, will make this a feature request... Thanks Buster ♫
